CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+#Contributing to Eclipse Collections
+
+Thanks for your interest in this project.
+
+## Project description:
+
+Eclipse Collections is a collections framework for Java. It has JDK-compatible List, Set and Map implementations with a rich API, additional types not found in the JDK like Bags, Multimaps and set of utility classes that work with any JDK compatible Collections, Arrays, Maps or Strings. The iteration protocol was inspired by the Smalltalk collection framework.
+
```

Comment:
These I brought from project description that we submit to Eclipse Foundation for project creation.  

It looks that README one is rather outdated as it doesn't mention Bags/Multimaps. Shall we replace README to match CONTRIBUTING/Wiki?